src/bin/common/solving_loop.ml

```diff
@@ -445,6 +445,13 @@ let main () =
       | ":random-seed"), _
       ->
       unsupported_opt name
+    (* Alt-Ergo custom options *)
+    | ":profiling", Symbol { name = Simple level; _ } ->
+      begin
+        match float_of_string_opt level with
+        | None -> print_wrn_opt ~name st_loc "nonnegative integer" value
```

Comment:
There needs to be a reply to the standard output here. This should be a `recoverable_error`.

Reply:
Actually no, this is a `set-option`, not a `get-option`, so it's OK